gears/gears-kotlin/src/main/kotlin/ApplyIf.kt

```diff
@@ -0,0 +1,54 @@
+package com.redmadrobot.gears.kotlin
+
+/**
+ * Applies the given block of modifications to the T if the condition is true.
+ *
+ * @param condition condition to determine if the block should be applied.
+ * @param block Lambda function that modifies the T.
+ * @return Modified T based on the condition.
```

Comment:
Specifying tags like `@param` and `@return`  for such simple functions is unnecessary. It might be clearer to write it in a single sentence:
```
Calls the specified function [block] with [this] value as its receiver if the [condition] is `true` and returns its result.
```
Simpler description — better.

Btw, I've just noticed that the semantics of the function `applyIf` differs from `apply`. Standard `apply` returns the receiver itself, while our function can return a modified result. It is more like `run` function semantics